app/server/pkg/community/google/analytics.go

```diff
@@ -0,0 +1,137 @@
+package google
+
+import (
+	"fmt"
+	"io/ioutil"
+	"net/http"
+	"strconv"
+	"time"
+
+	log "github.com/sirupsen/logrus"
+	"golang.org/x/oauth2"
+	"golang.org/x/oauth2/google"
+	"google.golang.org/api/analytics/v3"
+)
+
+const (
+	timeInterval = 30 * time.Minute
+	viewID       = "ga:208521052"
+	startDate    = "2019-12-01"
+	endDate      = "today"
+	filters      = "ga:eventCategory!=key1"
+	base         = 10
+	bitSize      = 64
+)
+type CommGAResponse struct{
+	TotalRuns string `json:"total-runs"`
+	OpInstalls string `json:"operator-installs"`
+	CityData [][]string `json:"geo-city"`
+	CountryData [][]string `json:"geo-country"`
+	DailyData [][]string `json:"daily-data"`
+}
+// GAResponseJSONObject is the array of GAResponse struct
+var GAResponseJSONObject CommGAResponse
+
+// Handler is responsible for the looping the UpdateAnalyticsData()
+func Handler() {
+	for true {
+		log.Infof("Updating Google Analytics Data ...")
+		err := getGraphData()
+		if err != nil {
+			log.Error(err)
+		}
+		err = getTotalCounts()
+		if err != nil {
+			log.Error(err)
+		}
+		log.Infof("Updated Google Analytics Data ...")
+		time.Sleep(timeInterval)
+	}
+}
+
+func getterJWTConfig() (*http.Client, error) {
+	key, err := ioutil.ReadFile("/etc/analytics/auth.json")
```

Comment:
Instead of re-define this method here, can't we use from the other packages

Reply:
I have kept this as a complete separate package as `community` because we will be shifting this later to repute, so I didn't want any external dependencies.

Reply:
okay!